test/cmdlineTests/standard_function_debug_info/output.json

```diff
@@ -0,0 +1,4 @@
+{"contracts":{"a.sol":{"A":{"evm":{"bytecode":{"functionDebugData":{}}}}}},"errors":[{"component":"general","errorCode":"3420","formattedMessage":"Warning: Source file does not specify required compiler version!
+--> a.sol
```

Comment:
I just realized that I could have used `--pretty-json` flag to make my `--combined-json` test output in tests look readable. I think we should make this flag work for Standard JSON too. Or is that already possible?

Not something to add in this PR of course.

Reply:
It is useful for combined-json because the output is not a single file. For standard-json you can just use `| jq`.

Reply:
True. You cannot use it in command-line tests though.